tests/utils.test.js

```diff
@@ -0,0 +1,28 @@
+const {convertToJson} = require("../scripts/utils")
+
+describe('convertToJson', () => {
+  test('should return JSON object if input is valid JSON string', () => {
+    const jsonString = '{"name": "AsyncAPI", "age": 5}';
+    const result = convertToJson(jsonString);
+    expect(result).toEqual({ name: "AsyncAPI", age: 5 });
+  });
+
+  test('should return JavaScript object if input is valid YAML string', () => {
+    const yamlString = 'name: AsyncAPI\nage: 5';
```

Comment:
Try to read actual `yaml` file here, and iff its not viable, use multiline string. Example:
```js
    const yamlString = `
      name: AsyncAPI
      age: 5
    `
```

Reply:
done